cmd/triggerrun/cmd/root_test.go

```diff
@@ -38,6 +39,40 @@ import (
 	rtesting "knative.dev/pkg/reconciler/testing"
 )
 
+func TestTrigger_Error(t *testing.T) {
+	//error case for show feature
+	buf := new(bytes.Buffer)
+	err := trigger("../testdata/trigger.yaml", "../testdata/http.txt", "show", buf)
+
+	wantError := "fail to get clients: Failed to build config from the flags: invalid configuration: no configuration has been provided"
+
+	if err.Error() != wantError {
+		t.Errorf("Error actual = %v, and Expected = %v.", err, wantError)
+	}
+
+	gotValue := buf.Bytes()
+	wantValue := []uint8(nil)
+	if diff := cmp.Diff(wantValue, gotValue); diff != "" {
+		t.Errorf("Trigger mismatch -want +got: %s", diff)
+	}
+
+	//error case for create feature
+	err = trigger("../testdata/trigger.yaml", "../testdata/http.txt", "create", buf)
```

Comment:
Its nice that we are trying to add test cases for both show and create. But in this case, the error that we are expecting happens before we every get to the switch statement for show or create:
```
	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
```

So, in effect, we are just testing the same case twice. So I'd recommend just keeping one of the cases instead of two.